maps/client.go

```diff
@@ -107,7 +107,12 @@ func (c *Client) FetchGoogleMapsLink(ctx context.Context, restaurant *gemini.Res
 			Rating:          float64(place.Rating),
 			UserRatingCount: int(*place.UserRatingCount),
 			GoogleMapsUrl:   fmt.Sprintf("https://www.google.com/maps/search/?api=1&query=xyz&query_place_id=%s", placeID),
-			Type:            place.PrimaryTypeDisplayName.Text,
+			Type: func() string {
```

Comment:
can you make sure primarytypedisplayname is not used anywhere and if so, remove it